### PR TITLE
{bio}[GCCcore/12.3.0] rnamotif v20240904

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0210u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8480CL (icelake), Python 3.6.8
See https://gist.github.com/branfosj/4704277343d81ed15989f0ef28c9645f for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0105u03a - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/43fcc3dd9c46521a6ea146a9df3c1704 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/branfosj/36c529561a0fc3b73f93f43402a385ac for a full test report.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node300.hydra.os - Linux Rocky Linux 8.10, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/smoors/c58b3913fdafb07e1340ed8142e85e5a for a full test report.

File: easybuild/easyconfigs/r/rnamotif/rnamotif-20240904-GCCcore-12.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+files_to_copy = [
+    (['src/rnamotif', 'src/rmprune', 'src/rmfmt', 'src/rm2ct'], 'bin'),
+    'LICENSE', 'efndata', 'Ecoli.trna.example',
+]
+
+sanity_check_paths = {
+    'files': ['bin/%(name)s'],
+    'dirs': ['efndata'],
+}
```
Comment (smoors) [inline easybuild/easyconfigs/r/rnamotif/rnamotif-20240904-GCCcore-12.3.0.eb:38]:
```suggestion
}

sanity_check_commands = ['rnamotif -h 2>&1 | grep usage']
```

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen3

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @branfosj!

**Pull request merged**